src/applications/edu-benefits/0994/pages/militaryService.js

```diff
@@ -1,11 +1,16 @@
 import fullSchema from 'vets-json-schema/dist/22-0994-schema.json';
-import { activeDutyNotice, benefitNotice } from '../content/militaryService';
+import {
+  activeDutyNotice,
+  benefitNotice,
+  reserveNationalGuardExpectedDutyTitle as selectedReserveNationalGuardExpectedDutyTitle,
```

Comment:
I'd remove this alias

Reply:
Thanks, yeah my IDE added that when I was renaming to a more specific variable name. I have pushed the change removing that